### PR TITLE
added some cisco devices, updated ME-3600X-24TS-M

### PR DESCRIPTION
**Pull request by jaannnis**
Added devices:
ME-3600X-24CX-M
N540-24Z8Q2C-M
ASR-920-12CZ-A
ASR-920-12SZ-IM

removed manufacturer from model:
ME-3600X-24TS-M

### CODE REVIEW

File: device-types/Cisco/ASR-920-12SZ-IM.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+is_full_depth: false
+u_height: 1
+comments: '[Cisco ASR 920 Series Aggregation Services Routers: High-Port-Density Models Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/asr-920-series-aggregation-services-router/datasheet-c78-733397.html)'
+power-ports:
+  - name: PSU0
+    type: A920-PWR400-A
+    maximum_draw: 400
+  - name: PSU1
+    type: A920-PWR400-A
+    maximum_draw: 400
```
Comment (DanSheps) [inline device-types/Cisco/ASR-920-12SZ-IM.txt:15]: These are not a valid type

File: device-types/Cisco/ME-3600X-24CX-M.txt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    type: 10gbase-x-sfpp
+  - name: Te0/3
+    type: 10gbase-x-sfpp
+  - name: Te0/4
+    type: 10gbase-x-sfpp
+power-ports:
+  - name: PSU1
+    type: PWR-ME3KX-AC
+  - name: PSU2
+    type: PWR-ME3KX-AC
```
Comment (DanSheps) [inline device-types/Cisco/ME-3600X-24CX-M.txt:75]: * ME PSU's are hot swappable and should be modelled as modules
* These are not a valid type

File: device-types/Cisco/N540-24Z8Q2C-M.txt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    type: 100gbase-x-qsfp28
+  - name: Hu 0/0/1/0
+    type: 100gbase-x-qsfp28
+power-ports:
+  - name: PSU0
+    type: N540-PWR400-A
+	maximum_draw: 400
+  - name: PSU1
+    type: N540-PWR400-A
+	maximum_draw: 400
```
Comment (DanSheps) [inline device-types/Cisco/N540-24Z8Q2C-M.txt:86]: * Not correct type
* These are hot-swappable and should be modelled as modules

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Do you still need this? @jaannnis

**Pull request closed without merging**